index.html

```diff
@@ -1,192 +1,192 @@
+                <p class="bio"> I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
 
-            <h2 class="connect">LET'S CONNECT</h2>
-            <ul class="icon-bio">
-                <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
-                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
-                <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
-                <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
-                <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
-            </ul>
-    </section>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul class="icon-bio">
+                    <li><a href="https://twitter.com/amiraldany"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/dany-augustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
+                    <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
+                    <li><a href="https://medium.com/@danyaugustin1982"><img src="img/medium.svg" alt="medium-icon"></a></li>
+                    <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
+                </ul>
+        </section>
 
-    <section class="Parent">
-        <article class="box">
-            <img class="image" src="img/tonic.png" alt="tonic">
-            <div class="box-info">
-                <h2 class="title">Tonic </h2>
-                <div class="description">
-                    <a href="#" class="client">CANOPY</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="tech">BACK END DEV</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="year">2015</a>
+        <section class="Parent">
+            <article class="box">
+                <img class="image" src="img/tonic.png" alt="tonic">
+                <div class="box-info">
+                    <h2 class="title">Tonic </h2>
+                    <div class="description">
+                        <a href="#" class="client">CANOPY</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="tech">BACK END DEV</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="year">2015</a>
+                    </div>
```

Comment:
- [ ] :clap: Creativity at its peak! I will only suggest that since this is more like a list of items with similar attributes, you can use a `ul`/`li` tag to enclose the `a` tags and the `img` tags. How about that :grey_question: